guide/english/html/attributes/input/index.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Input
 ---
-## Input
-The HTML `<input>` tag is used within a form to declare an input element.
-It allows the user to enter data.
+
+# Input
```

Comment:
Please change the heading back to `##`. 
You can use this page for reference: https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/css/borders/index.md. Click on the <kbd>Raw</kbd> button if you want to see its markdown.